codesy/templates/401.html

```diff
@@ -1 +1,13 @@
-<a href="https://{{ current_site.domain }}">Sign in to codesy.io</a>.
+{% load staticfiles %}
+<link rel="stylesheet" href="//{{current_site.domain}}{% static "css/styles.css" %}">
+<link rel="stylesheet" href="//cdn.jsdelivr.net/pure/0.5.0/pure-min.css">
```

Comment:
Yeah, that makes it a little tricky. Go ahead and file the issue and I'll merge this.